base/14.16.0/Dockerfile

```diff
@@ -34,27 +34,24 @@ RUN apt-get update && \
   ttf-wqy-microhei \
   xfonts-wqy \
   # clean up
+  && apt-get clean \
```

Comment:
apparently, not needed
`docker run --rm node:14.16.0-buster cat /etc/apt/apt.conf.d/docker-clean`

Reply:
Overall, I trust `apt-get clean` to do the cleanup if a previous layer messes it up, rather than the current manual steps :-)

I'll leave this here :+1: